xc/common/utils/prjxray_create_place_constraints.py

```diff
@@ -493,39 +462,17 @@ def get_tile_capacities(arch_xml_filename):
     return grid
 
 
-def get_vpr_coords_from_site_name(conn, site_name, grid_capacities):
+def get_vpr_coords_from_site_name(
+        site_dict, tile_dict, site_name, grid_capacities
```

Comment:
Rather than passing around all these dict's, please make a container with named methods to access the underlying data.

Reply:
Done